pkgs/build-support/build-fhs-userenv/env.nix

```diff
@@ -68,60 +68,10 @@ let
     ${profile}
   '';
 
-  # Compose /etc for the chroot environment
-  etcPkg = stdenv.mkDerivation {
```

Comment:
The only reason I can think of is to provide `/etc/ld.so.conf`

Reply:
cc @vcunat @matthewbauer Any idea why this was added in the first place? Is there some impurity introduced if we keep the original `/etc`?

Reply:
Is this maybe needed on non-nixos distributions because `/etc/ld.so.conf` could mess with the build?

Reply:
My guess of the intention: link as little as possible, thus reducing impurity brought from the host OS.